src/models/TestResultsDAO.ts

```diff
@@ -236,51 +236,6 @@ export class TestResultsDAO {
     // });
   }
 
-  public getActivity(filters: {
```

Comment:
Was this method just never used?

Reply:
Because it's only used by the `checkTestTypeStartAndEndTimestamp` function, and we're removing the functionality to check the test being amended fits within an activity, just thought I'd cleanup